exercises/practice/yacht/.meta/solutions/yacht.rb

```diff
@@ -0,0 +1,77 @@
+class Yacht
```

Comment:
This file should be `.meta/example.rb` and the path in `config.json` should be updated too. I'll fix the issue that's causing this.

Reply:
FYI, I've fixed the generating code here for future exercises: https://github.com/exercism/ruby/pull/1628